src/Infrastructure/OrchestratorService.cs

```diff
@@ -46,6 +46,17 @@ public async Task FetchMapAndSaveEmployeeData()
 
         var phoneNumberUtil = PhoneNumberUtil.GetInstance();
 
+        foreach (var bemanning in bemanningEntries.Where(employee => !IsActiveEmployee(employee)))
+        {
+            // Remove potential employees that shouldn't have been added.
+            // This should normally not happen, but might happen in cases where
+            // StartDate in bemanning wasn't set properly when orchestrating.
+            _logger.LogInformation(
+                "Deleting employee with email {BemanningEmail} from database, since they haven't started yet",
+                bemanning.Email);
+            await _employeesRepository.EnsureEmployeeIsDeleted(bemanning.Email);
+        }
+

```

Comment:
Skal denne koden kjøres bare én gang? Isåfall må den jo ryddes opp i i etterkant.

Reply:
Nei, egentlig ikke. Det kan fort skje i periodisk kjøring av orkestrering. Det kan skje at du legger til en ny ansatt i bemanning som ikke får satt prosjektet `Ikke startet` før chewbacca-orkestrering kjører. Da vil du komme i tilsvarende tilstand. Så i de fleste tilfeller vil dette kjøre bare for å slette brukere som ikke lengre er aktive og slette det.

Teknisk sett kan det også kjøre ved sluttdato for enkelte og fjerne de helt. Det kan jo være uønsket, men også ønsket egentlig. Mulig vi også skulle ha slettet blob i så fall